proxy/mempool/mempool_periodic_task_gas_price.py

```diff
@@ -60,5 +64,22 @@ def _process_error(self, _: MPGasPriceRequest) -> None:
         pass
 
     async def _process_result(self, _: MPGasPriceRequest, mp_res: MPGasPriceResult) -> None:
+        for token_list in mp_res.token_list:
+            chain_id_token_name = f'{token_list.chain_id}:{token_list.token_name}'
```

Comment:
Each chain-id has only one payment token.
So, it is enough to store prices in a dictionary with a key by `chain-id` or `token-name`:
```
self._min_executable_gas_prices[token_list.chain_id] = list()
```

Reply:
Oh, interesting. Thanks I didn't notice this.